.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+Resolves [#ISSUE]
+
+## General
+
+- [ ] The PR should not contain any spurious changes, i.e. changes that do not relate to the issue linked above
+
+## If the PR involves changes to annotation properties
+
+- [ ] tbd
+
+## If the PR involve changes to the logic (isa, equivalent class)
+
+- [ ] If an equivalent class axiom is added, a "dc:conformsTo" annotation needs to be added linking to the corresponding design pattern
+- [ ] If a class is changed that has any indirect children, associated with them (more than two levels of classes underneath), 
+      it must be reviewed by member of core Uberon team
+
```

Comment:
```suggestion
- [ ] #gogoeditdiff is run
```

Reply:
Will adding this comment trigger the gogoeditdiff?